pkg/network/tracer/tracer_usm_linux_test.go

```diff
@@ -211,6 +212,39 @@ func TestHTTPSViaLibraryIntegration(t *testing.T) {
 	}
 }
 
+func buildPrefetchFileBin(t *testing.T) string {
+	const srcPath = "prefetch_file"
+	const binaryPath = "testutil/prefetch_file/prefetch_file"
+
+	t.Helper()
+
+	cur, err := testutil.CurDir()
+	require.NoError(t, err)
+
+	binary := fmt.Sprintf("%s/%s", cur, binaryPath)
+	// If there is a compiled binary already, skip the compilation.
+	// Meant for the CI.
+	if _, err = os.Stat(binary); err == nil {
+		return binary
+	}
+
+	srcDir := fmt.Sprintf("%s/testutil/%s", cur, srcPath)
+
+	c := exec.Command("go", "build", "-buildvcs=false", "-a", "-ldflags=-extldflags '-static'", "-o", binary, srcDir)
+	out, err := c.CombinedOutput()
+	t.Log(c, string(out))
+	require.NoError(t, err, "could not build  test binary: %s\noutput: %s", err, string(out))
+
+	return binary
+}
+
+func prefetchLib(t *testing.T, filename string) *exec.Cmd {
+	prefetchBin := buildPrefetchFileBin(t)
+	cmd := exec.Command(prefetchBin, filename, "5")
```

Comment:
Wouldn't this be treated as 5 nanoseconds by `time.Duration`? You should be able to use `5s` here as that should be parsed correctly by `time.Duration`.